### PR TITLE
Add LN-DLC to DLC channels pivot blog post

### PR DESCRIPTION
**Pull request by luckysori**
Fixes https://github.com/get10101/meta/issues/288.

### CODE REVIEW

File: blog/2024-02-09-see-you-later-mr-lightning.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+---
+
+<!-- LTeX: SETTINGS language=en-GB-->
+
+If you are reading this post, you may have already heard that we have closed all Lightning channels with our users, replacing them with DLC channels.
+That may sound like a step backwards, so we want to explain our reasoning behind this decision and tell you why we think it is actually a good thing.
+
+## Why we started with Lightning
+
+10101 was conceived as a Lightning mobile wallet with trading capabilities.
```
Comment (bonomat) [inline blog/2024-02-09-see-you-later-mr-lightning.md:15]: ? 

```suggestion
10101 was conceived as a mobile Lightning wallet with trading capabilities.
```

File: blog/2024-02-09-see-you-later-mr-lightning.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+We've had problems with Lightning at every step of the user's journey through 10101:
+
+- Users found it difficult to move coins into 10101.
+A Lightning payment can fail for many reasons, and it is often hard to pinpoint the cause.
+We generally overcame these issues as we learnt more about Lightning, but it is pretty scary to lose a potential user at the first hurdle.
+
+- Users found it very difficult to move coins out of 10101.
+If you have used 10101 to pay an invoice, you have almost certainly run into the dreaded `route not found` error message.
+We were working on solutions to this for
```
Comment (bonomat) [inline blog/2024-02-09-see-you-later-mr-lightning.md:36]: Sentence seems to be incomplete.

File: blog/2024-02-09-see-you-later-mr-lightning.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+A naive approach would be for the user to send us some sats plus a Lightning invoice, and for us to pay the invoice once the payment in our DLC channel has been committed.
+This has the obvious downside that the user has to trust us to make the payment on their behalf.
+Nevertheless, it could be acceptable for small payments.
+
+To make it "trustless", we would have to make the payment within the DLC channel conditional on the Lightning payment being claimed.
+The user would generate a Lightning invoice at the destination wallet and feed it into the 10101 app.
+The app would then add a HTLC to the DLC channel, locked against the secret hash of the invoice.
+Our 10101 node would then make the payment on Lightning, which, once claimed by the destination wallet, would reveal the secret back to us.
+With the secret we would then be able to claim the HTLC in the DLC channel, but we would then collaborate with the app to remove the HTLC altogether and update the balances accordingly.
```
Comment (bonomat) [inline blog/2024-02-09-see-you-later-mr-lightning.md:81]: Should we add a diagram for this?

File: blog/2024-02-09-see-you-later-mr-lightning.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+Obviously, that just sounds like Lightning.
+But in reality the payment within the DLC channel could be unlocked by a secret that has nothing to do with a Lightning payment.
+Which means that the DLC channel can be oblivious of Lightning altogether, simplifying the implementation.
+And the app does not need to run a Lightning node!
+
+## Outro
+
+On a personal note, I am proud of the team for being humble and admitting that we had to change our approach.
+As with most things in life, we now wish we had done it sooner, but hindsight is always 20/20.
```
Comment (bonomat) [inline blog/2024-02-09-see-you-later-mr-lightning.md:91]: > but hindsight is always 20/20.

TIL 😅

File: blog/2024-02-09-see-you-later-mr-lightning.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+If you have any further questions or comments, please reach out on Telegram (https://t.me/get10101/1) or on [Twitter](http://twitter.com/).
```
Comment (bonomat) [inline blog/2024-02-09-see-you-later-mr-lightning.md:93]:
```suggestion
If you have any further questions or comments, please reach out on [Telegram](https://t.me/get10101/1) or on [Twitter](http://twitter.com/get10101). 
If you haven't used our app yet, make sure to had over to [10101.finance](https://10101.finance) and find the download links to your favorite app store. 
```

File: static/index.html [hunk 6]
```diff
@@ -673,7 +673,7 @@ <h2 class="pb-4">Trade without counterparty risk.</h2>
               <div class="section-heading text-center">
                 <h2>Learn More!</h2>
                 <a
-                  href="http://10101.finance/blog"
```
Comment (bonomat) [inline static/index.html:676]: 👍

**Review by bonomat — APPROVED** [review]
Great write up. Thank you for this.

File: blog/2024-02-09-see-you-later-mr-lightning.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+- Users found it very difficult to move coins out of 10101.
+If you have used 10101 to pay an invoice, you have almost certainly run into the dreaded `route not found` error message.
+We were working on solutions to this for
+
+- Channels would be force-closed for no apparent reason.
+Of course, there is always a reason.
+Disagreements during fee negotiations and expired HTLCs were the main culprits here.
+Regardless, any unnecessary force-closure is catastrophic because it is a waste of money and time for everyone involved.
+And this is particularly egregious when you are in a high transaction fee environment, like the one we experienced in 2023.
+
```
Comment (holzeis) [inline blog/2024-02-09-see-you-later-mr-lightning.md:43]: Its probably worthwhile to add the huge on-chain footprint and complexity coming from an embedded dlc channel. The key promise of simply force closing a channel and you get your funds back wasn't that present anymore given high fee environments, etc.

**Comment by bonomat** [comment]
Awesome work @luckysori , let's publish this.

**Pull request merged**